packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1141,6 +1149,16 @@ abstract class _RenderCustomClip<T> extends RenderProxyBox {
   T get _defaultClip;
   T _clip;
 
+  /// {@template flutter.widget.clipper.clipBehavior}
+  /// Controls how to clip when clipper is null. Otherwise, clipper controls the
+  /// clip behavior.
+  ///
+  /// Default to [Clip.antiAlias]. [Clip.none] is not allowed here.
+  /// {@endtemplate}
+  Clip _clipBehavior;
+
+  Clip get clipBehavior => clipper == null ? _clipBehavior : clipper.clipBehavior;
```

Comment:
I think we should just expose the _clipBehavior directly, I don't see any reason to have a getter that defers to `clipper` sometimes.